cmd/api/src/database/audit.go

```diff
@@ -78,5 +81,27 @@ func (s *BloodhoundDB) ListAuditLogs(before, after time.Time, offset, limit int,
 		result = cursor.Find(&auditLogs).Count(&count)
 	}
 
+	// Populate the actor name and email from the users table. Uses a map as a cache to avoid looking up the same user multiple times.
```

Comment:
We should probably do a `CheckError(result)` after running the first db query, since it doesn't make sense to continue if we already had an error.